test/widgets/lightbox_test.dart

```diff
@@ -236,14 +236,19 @@ void main() {
       debugNetworkImageHttpClientProvider = null;
     });
 
-    testWidgets('app bar shows sender name and date', (tester) async {
+    testWidgets('app bar shows sender avatar, name and date', (tester) async {
       prepareBoringImageHttpClient();
       final timestamp = DateTime.parse("2024-07-23 23:12:24").millisecondsSinceEpoch ~/ 1000;
       final message = eg.streamMessage(sender: eg.otherUser, timestamp: timestamp);
       await setupPage(tester, message: message, thumbnailUrl: null);
 
-      // We're looking for a RichText, in the app bar, with both the
-      // sender's name and the timestamp.
+      // Check Avatar properties
+      final avatar = tester.widget<Avatar>(find.byType(Avatar));
+      check(avatar.size).equals(36);
+      check(avatar.borderRadius).equals(36 / 8);
```

Comment:
The design properties like `size` and `borderRadius` are usually verified during reviews. We typically don't need test for these because it only captures a limited scope of bugs and becomes another place to update if the design spec changes.

Reply:
Thank you for the review! Should I remove the test, or would you suggest modifying it in some way? Looking forward to your guidance!